src/react_native/reanimated.cljs

```diff
@@ -1,6 +1,7 @@
 (ns react-native.reanimated
   (:require ["react-native" :as rn]
             ["react-native-linear-gradient" :default LinearGradient]
+            ["@react-native-community/blur" :as blur]
```

Comment:
we have react-native.blur

Reply:
@OmarBasem has been working on scroll page animations on pr #14695 . This pr add sticky tabs based on his work. I think he might give a better reply on why `@react-native-community/blur` in this case

Reply:
yeah i know,  we should use `react-native.blur`